### PR TITLE
Adds pagerfanta paginator everywhere, modifies article routing.

### PR DESCRIPTION
**Pull request by frankoa**
Adds pagerfanta paginator everywhere, and modifies article routing. 
Change API for is_starred and is_archived. (Reflects the output of the API 'getEntries')

### CODE REVIEW

File: src/Wallabag/CoreBundle/Repository/EntryRepository.php [hunk 1]
```diff
@@ -13,77 +13,66 @@ class EntryRepository extends EntityRepository
      * Retrieves unread entries for a user.
      *
      * @param int $userId
-     * @param int $firstResult
-     * @param int $maxResults
      *
      * @return Paginator
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Repository/EntryRepository.php]: Could you adjust the docblock for every function? It now returns a `DoctrineORMAdapter`.
Also, you can remove the `use Doctrine\ORM\Tools\Pagination\Paginator;`

**Comment by j0k3r** [comment]
Sadly test are failing but it seems to be unrelated to your change.
Can you take some time to see why they are failing? Thanks :)

**Comment by frankoa** [comment]
@j0k3r Ok, I'll check tests today.

**Comment by frankoa** [comment]
It was related to my changes.
Pagination adds some conflicts between routes. 
(archive/1 for page 1 and archive/1 for "add entry 1 to archive")

I modified routing : 
- archive/list/3 loads 3rd page of archived entries
- archive/3 toggle archive flag on entry with id 3

Let me know if you want me to change routing.

**Comment by j0k3r** [comment]
Great thanks @frankoa :+1:

**Pull request merged**